galley/pkg/crd/validation/webhook_test.go

```diff
@@ -312,6 +323,16 @@ func TestAdmitPilot(t *testing.T) {
 			},
 			allowed: false,
 		},
+		{
+			name:  "invalid extra key create",
+			admit: wh.admitPilot,
+			in: &admissionv1beta1.AdmissionRequest{
+				Kind:      metav1.GroupVersionKind{}, // TODO
```

Comment:
Please set the right GVK.

Reply:
@ayj need advice.  Every subtest in "TestAdmitPilot()" specifies `Kind:      metav1.GroupVersionKind{}, // TODO`.  I followed the pattern; every subtest in "TestAdmitMixer() specifies `Kind:      metav1.GroupVersionKind{Kind: "mock"},`.  TestAdmitPilot is happy to take `Kind:"mock"`.  What was the intention of the TODOs?

Reply:
We can replace the TODO with a mock GVK variable, e.g.

```golang
mockGVK := metav1.GroupVersionKind{ Kind: "mock" }
```

As you noted, the kind isn't used by the function under test.